doc/source/intro.rst

```diff
@@ -0,0 +1,136 @@
+policies| and |metrics|, along with their definitions, types and properties.
+The indexer is also responsible for linking |resources| with |metrics|.
+
+.. _PostgreSQL: http://postgresql.org
+.. _MySQL: http://mysql.org
+
+Architecture overview
+---------------------
+
+Gnocchi consists of several services: a HTTP REST API (see :doc:`rest`), an
+optional statsd-compatible daemon (see :doc:`statsd`), and an asynchronous
+processing daemon (named `gnocchi-metricd`). Data is received via the HTTP REST
+API or statsd daemon. `gnocchi-metricd` performs operations (statistics
+computing, |metric| cleanup, etc...) on the received data in the background.
+
+.. image:: _static/architecture.svg
+  :align: center
+  :width: 95%
+  :alt: Gnocchi architecture
+
+.. image source: https://docs.google.com/drawings/d/1aHV86TPNFt7FlCLEjsTvV9FWoFYxXCaQOzfg7NdXVwM/edit?usp=sharing
+
+All those services are stateless and therefore horizontally scalable. Contrary
+to many time series databases, there is no limit on the number of
+`gnocchi-metricd` daemons or `gnocchi-api` endpoints that you can run. If your
+load starts to increase, you just need to spawn more daemons to handle the flow
+of new requests. The same applies if you want to handle high-availability
+scenarios: just start more Gnocchi daemons on independent servers.
+
+
+Understanding aggregation
+-------------------------
+
+The way data points are aggregated is configurable on a per-metric basis, using
+an archive policy.
+
+An archive policy defines which aggregations to compute and how many aggregates
+to keep. Gnocchi supports a wild number of aggregation methods, such as
+minimum, maximum, average, Nth percentile, standard deviation, etc. Those
+aggregations are computed over a period of time (called granularity) and are
+kept for a defined timespan. Aggregates are stored in a compressed format,
+ensuring the data points take as little space as possible.
+
+For example, imagine you define an archive policy to keep the average, minimum,
+and maximum of a |time series| with five-minute granularity for 30 days. In
+that case, Gnocchi will compute the average, minimum, and maximum of the values
+over five-minute ranges, keeping up to 8,640 points (the number of five-minute
+aggregates you can have over 30 days).
+
+How to define archive policies
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The |archive policies| define how the |metrics| are aggregated and how long
+they are stored. Each |archive policy| definition is expressed as the number of
+points over a |timespan|.
+
+If your |archive policy| defines a policy of 10 points with a |granularity| of
+1 second, the |time series| archive will keep up to 10 seconds, each
+representing an aggregation over 1 second. This means the |time series| will at
+maximum retain 10 seconds of data (sometimes a bit more) between the more
+recent point and the oldest point. That does not mean it will be 10 consecutive
+seconds: there might be a gap if data is fed irregularly.
```

Comment:
this is basically a duplicate of above "For example, imagine..." paragraph. i think maybe we should just keep one to minimise amount of words

Reply:
yup, removed above

Reply:
gonna move that paragraph in operating back then